puppet/modules/freight/files/archivedeb-HEADER.html

```diff
@@ -18,9 +18,11 @@ <h1>The Debian Foreman archive repo</h1>
 
 <p>An example of how to add it:</p>
 
-<pre>wget https://archivedeb.theforeman.org/foreman.asc -O - | apt-key add
+<pre>
+wget https://archivedeb.theforeman.org/foreman.asc -O - | apt-key add
 echo "deb http://archivedeb.theforeman.org/ buster nightly" > /etc/apt/sources.list.d/foreman.list
-echo "deb http://archivedeb.theforeman.org/ plugins nightly" >> /etc/apt/sources.list.d/foreman.list</pre>
+echo "deb http://archivedeb.theforeman.org/ plugins nightly" >> /etc/apt/sources.list.d/foreman.list
```

Comment:
I doubt we'll ever have `nightly` in our archive. Perhaps `MAJOR.MINOR` is a better choice?

Reply:
haha, indeed